documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -9,243 +9,204 @@ You may obtain a copy of the License at
+the application ends, the Pravega resources will be properly closed in the
 right order.
 
-Now that we have a ClientFactory, we can use it to create a Writer.  There are
-several things a developer needs to know before s/he creates a Writer:
+Once the `ClientFactory` is instantiated, we can use it to create a Writer. There are
+several things a developer needs to know before creating a Writer:
+
+1.  _What is the name of the Stream to write to?_
+    (The Scope has already
+    been determined when the `ClientFactory` was created.)
 
-1.  What is the name of the Stream to write to?  Note: the Scope has already
-    been determined when the ClientFactory was created
+2.  _What Type of Event objects will be written to the Stream?_
 
-2.  What Type of Event objects will be written to the Stream?
+3.  _What serializer will be used to convert an Event object to bytes?_ (Recall
+    that Pravega only knows about sequences of bytes, it is unaware about Java objects.)
 
-3.  What serializer will be used to convert an Event object to bytes?  Recall
-    that Pravega only knows about sequences of bytes, it does not really know
-    anything about Java objects.
+4.  _Does the Writer need to be configured with any special behavior?_
 
-4.  Does the Writer need to be configured with any special behavior?
 
-In our example, lines 11-13 show all these decisions.  This Writer writes to the
-Stream specified in the configuration of the HelloWorldWriter object itself (by
-default the stream is named "helloStream" in the "examples" Scope).  The Writer
+```Java
+     EventStreamWriter<String> writer = clientFactory.createEventWriter(streamName,
+                                                      new JavaSerializer<String>(),
+                                                      EventWriterConfig.builder().build()))
+```
+The `EventStreamWriter` writes to the
+Stream specified in the configuration of the `HelloWorldWriter` sample application (by
+default the stream is named "helloStream" in the "examples" Scope). The Writer
 processes Java String objects as Events and uses the built in Java serializer
 for Strings.  
 
-The EventWriterConfig allows the developer to specify things like the number of
-attempts to retry a request before giving up and associated exponential back
-settings.  Pravega takes care to retry requests in the case where connection
-failures or Pravega component outages may temporarily prevent a request from
+The `EventWriterConfig` allows the developer to specify things like the number of
+attempts to retry a request before giving up and associated exponential back-off
+settings. Pravega takes care to retry requests in the presence of connection
+failures or Pravega component outages, which may temporarily prevent a request from
 succeeding, so application logic doesn't need to be complicated with dealing
```

Comment:
`...be complicated with dealing with intermittent cluster failures.` -> ...be complicated (by) dealing with...

Reply:
Accepted suggestion